api_spec.yaml

```diff
@@ -1231,11 +1231,217 @@ paths:
+                                                      type: object
+                                                      properties:
+                                                          enabled:
+                                                              type: boolean
+                                                  thirdParty:
+                                                      type: object
+                                                      properties:
+                                                          enabled:
+                                                              type: boolean
+                                                          providers:
+                                                              type: array
+                                                              items:
+                                                                  $ref: "#/components/schemas/thirdPartyProvider"
+                                                  firstFactors:
+                                                      type: array
+                                                      items:
+                                                          type: string
+                                                      example:
+                                                          - "emailpassword"
+                                                          - "thirdparty"
+                                                  requiredSecondaryFactors:
+                                                      type: array
+                                                      items:
+                                                          type: string
+                                                      example:
+                                                          - "otp-email"
+                                                  coreConfig:
+                                                      type: object
+                                                  userCount:
+                                                      type: number
+                                                  validFirstFactors:
+                                                      type: array
+                                                      items:
+                                                          type: string
+                                                      example:
+                                                          - "emailpassword"
+                                                          - "thirdparty"
+                                                      description: The first factors that the tenant can use based on the recipes initialized and login methods enabled
+                                                  mergedProvidersFromCoreAndStatic:
+                                                      type: array
+                                                      items:
```

Comment:
do we need provider list and this? also, if it is required, can this be moved under thirdParty only?

Reply:
Yes we need both of them because of the logic in frontend that figures what providers need to be added to the core.